ui/DesignSystem/Component/Notification.svelte

```diff
@@ -1,67 +1,72 @@
 <script>
-  import { Icon, Title } from "../Primitive";
+  import { Icon, Text, Title } from "../Primitive";
 
-  export let icon = Icon.Info;
   export let style = null;
-
   export let level = "info";
+  export let noIcon = false;
```

Comment:
I think avoiding a double negative makes it more readable and also make the "action" configurable (as per styleguide):
```suggestion
  export let showIcon = true;
  export let action = "Close";
```

You'd have to also adjust the `if` branch below for this to work and also use the `action` variable in the template like so:
```
<Title>{action}</Title>
```

Reply:
@rudolfs We wanted to keep this PR simple for now and have the same functionality as before. Adding custom actions here will need some more consideration as we will have to find a way of passing the actual actions to the notification faucet, eg repeat a search query.

Reply:
Leaving the "action" stuff out for now. But address the double negative.